e2e/node_config.go

```diff
@@ -381,7 +381,6 @@ func baseTemplateLinux(location string, k8sVersion string) *datamodel.NodeBootst
 			"--anonymous-auth":                    "false",
 			"--authentication-token-webhook":      "true",
 			"--authorization-mode":                "Webhook",
-			"--azure-container-registry-config":   "/etc/kubernetes/azure.json",
```

Comment:
should we make this conditional on the k8s version (1.30 should be the pivot I think), and also do the same for windows? (in this same file)

Reply:
Where are we actually setting the k8s version?